packages/web-components/src/accordion-item/README.md

```diff
@@ -0,0 +1,142 @@
+| `size`               | size               |
+
+<br />
+
+### **Slots**
+
+| Name             | Description                                                                      |
+| ---------------- | -------------------------------------------------------------------------------- |
+| `start`          | The slot used for a presentation icon when expanded                              |
+| `end`            | The slot used for a presentation icon when collapsed                             |
+| `heading`        | Content which serves as the accordion item heading and text of the expand button |
+|                  | The default slot for accordion item content                                      |
+| `expanded-icon`  | The slot used for a custom expanded icon                                         |
+| `collapsed-icon` | The slot used for a custom collapsed icon                                        |
+
+<br />
+<hr />
+<br />
+
+## **Accessibility**
+
+[W3 Accordion Item Spec](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/)
+
+<br />
+
+### **WAI-ARIA Roles, States, and Properties**
+
+- `role = "button"`
+  - The title of each accordion header is contained in an element with role button.
+- `role = "heading"`
+  - Each accordion header button is wrapped in an element with role heading that has a value set for aria-level that is appropriate for the information architecture of the page.
+- `aria-expanded`
+  - If the accordion panel associated with an accordion header is visible, the header button element has aria-expanded set to true. If the panel is not visible, aria-expanded is set to false.
+- `aria-controls`
+  - The accordion header button element has aria-controls set to the ID of the element containing the accordion panel content.
+- `aria-disabled`
+  - If the accordion panel associated with an accordion header is visible, and if the accordion does not permit the panel to be collapsed, the header button element has aria-disabled set to true.
+
+<br />
+<hr />
+<br />
+
+## **Preparation**
+
+<br />
+
+### **Fluent Web Component v3 v.s Fluent React 9**
+
+<br />
+
+**Component and Slot Mapping**
+
+| Fluent UI React 9   | Fluent Web Components 3   |
+| ------------------- | ------------------------- |
+| `<AccordionItem>`   | `<fluent-accordion-item>` |
+| `<AccordionHeader>` | `named slot = "heading"`  |
+| `<AccordionPanel>`  | `default slotted content` |
+
+<br />
+
+**Property Mapping**
+| Fluent UI React 9 | Fluent Web Components 3 | Description of difference |
+|---------------------------|---------------------------|------------------------------------------------------------------------------------------|
+| `defaultOpenItems: number`| `expand: boolean` | _FuiR9_ `defaultOpenItems` is a number property set on the `Accordion` corresponding to the intended `AccordionItem` to be expanded.<hr /> `expand` is a boolean property set directly on the `AccordionItem` intended to be expanded.
```

Comment:
I would expect `defaultOpenItems` to be documented both here in `accordion-item` and in `accordion` as well.